testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Kafka.java

```diff
@@ -50,6 +60,13 @@ public void extendEnvironment(Environment.Builder builder)
         builder.addContainers(createZookeeper(), createKafka(), createSchemaRegistry())
                 .containerDependsOn(KAFKA, ZOOKEEPER)
                 .containerDependsOn(SCHEMA_REGISTRY, KAFKA);
+
+        MountableFile logConfigFile = forHostPath(configDir.getPath("log.properties"));
+        builder.configureContainers(container -> {
+            if (isPrestoContainer(container.getLogicalName())) {
```

Comment:
would like to rename `isPrestoContainer` to `isTrinoContainer` as separate PR?

Reply:
Created https://github.com/trinodb/trino/pull/14195